common/config/src/lib.rs

```diff
@@ -465,7 +465,7 @@ pub fn get_hostname(alt: Option<&[&dyn AsRef<OsStr>]>) -> Option<String> {
             };
             match File::open(path).and_then(closure) {
                 Ok(s) => return Some(s.trim().to_string()),
-                Err(e) => warn!("Unable to read hostname from {:?}: {}", path, e),
+                Err(e) => warn!("Unable to read hostname from {path:?}: {e}"),
```

Comment:
This is personal, but I prefer to keep the variables outside of the message. I know there's a clippy lint that complains about this.

I disable the clippy lint:

```
[lints.clippy]
uninlined-format-args = { level = "allow", priority = 127 }
```

Why? `{foo:?}` works, but `{bar.quz:?}` does not work, so when you're dotting you get `warn!("something {foo:?} something else {:?}", bar.quz);` which is confusing. 

Ideally we move the parameters outside of the message, like so:

```
warn!(?err = e, path = %path.display(), "Unable to read hostname");
```

Reply:
Totally fair. Resolving.